src/shared/constants/api.ts

```diff
@@ -0,0 +1 @@
+export const MOVIES_BASE_URL = 'http://localhost:3001';
```

Comment:
Let's take this value from env variable 
export const MOVIES_API_URL = process.env.MOVIES_API_URL || 'http://localhost:3001';
and don't forget to add it to your .env file :D